include/core/adapter/Adapter2D.hpp

```diff
@@ -0,0 +1,69 @@
+#ifndef GENERIC_ADAPTER_H
+#define GENERIC_ADAPTER_H
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+#include <boost/geometry.hpp>
+#include <boost/geometry/geometries/geometries.hpp>
+#include <HY_Catchment.hpp>
+#include <HY_HydroLocation.hpp>
+
+using namespace hy_features::hydrolocation;
```

Comment:
Am I missing a reason why the _HY_Catchment.hpp_ and _HY_HydroLocation.hpp_ are being imported, and why the _hy_features::hydrolocation_ namespace is being used?  They don't seem to hurt anything, but I am curious (especially given the elsewhere mentioned name discrepancy) if any intended changes could have been accidentally left out.

Reply:
They are imported because this class was made by modifing another and the includes where not removed, Another thing in the branch to correct.